routes/index.js

```diff
@@ -327,6 +327,11 @@ module.exports = function (passport) {
 
   });
 
```

Comment:
I think you want this to be a router.get request. I swapped that out and was able to get a list of valid users for a station(s). I subsequently added/removed privileges to my account and it reflected the changes accordingly.